README.md

```diff
@@ -132,6 +132,8 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Habitat supervisor during the install.
 
+*Important*: If you get authentication failures with a self-signed cert, you may need to either update the cert package that the habitat services are using (in the `/etc/systemd/system/hab-sup.service` file), or modify the hab `core/cacerts` package (in the `/hab/pkgs/core/cacerts/...` folder) to add your self-signed cert chain to the `cert.pem` file. Restart the hab services with `systemctl restart hab-sup` after updating the cert file.
```

Comment:
Alternatively, you can copy the cacerts.pem, append your cert to it and save that file into a file location where root can get to. 

Then when you provision the hab-sup make sure to define `SSL_CERT_FILE` to the path location.  Systemd will set that env var to that cert bundle and any services that hab-sup loads will use that.